source/MapOutfitterPanel.cpp

```diff
@@ -193,7 +193,7 @@ void MapOutfitterPanel::DrawItems()
 			string price = Format::Credits(outfit->Cost()) + " credits";
 			
 			string info;
-			if(outfit->Get("installable") < 0.)
+			if(outfit->Get("minable") > 0.)
```

Comment:
Do we still want a line like `(This outfit is uninstallable)` or something for uninstallable outfits? I think it makes sense.

Reply:
I don't think the tag on its own is super helpful, since an uninstallable outfit can still be (and is primarily meant to) be stored in cargo. However, it will say that they take up "0 tons of outfit space", so swapping that to "<x> tons of <mass>" seems like the way to go.

Reply:
Uninstallable outfits will now refer to their mass on the outfitter map catalog.